packages/block-editor/src/components/button-block-appender/index.js

```diff
@@ -18,14 +18,15 @@ import deprecated from '@wordpress/deprecated';
 import Inserter from '../inserter';
 
 function ButtonBlockAppender(
-	{ rootClientId, className, onFocus, tabIndex },
+	{ rootClientId, className, onFocus, tabIndex, onSelectCallback },
 	ref
 ) {
 	return (
 		<Inserter
 			position="bottom center"
 			rootClientId={ rootClientId }
 			__experimentalIsQuick
+			onSelectCallback={ onSelectCallback }
```

Comment:
Is the `Callback` suffix necessary? The `on` prefix already hints that it's a function so I'd simply call this `onSelect`.